### DISCUSSION TITLE
Can I customize the floater arrow color while using a custom tooltipComponent?

### DISCUSSION THREAD

**Discussion by aaron-parisi-turngate** [gilbarbara/react-joyride#1116] [Q&A] [answered]
I know I can use a [custom component](https://docs.react-joyride.com/custom-components) and am doing that successfully.

I also am able to change the length of the arrow via:

```
				floaterProps={{
					styles: {
						arrow: {
							length: 100,
							fill: "#ff0000",
						},
					},
				}}
```

But the `fill` attribute isn't having any impact.  Am I missing something?

**Comment by gilbarbara** [comment, marked as answer]
Hey @aaron-parisi-turngate 

It's in the [docs](https://docs.react-joyride.com/styling)
